access.md

```diff
@@ -15,32 +15,15 @@ The serial console of a virtual machine can be accessed by using the `console`
 command:
 
 ```bash
-$ virtctl console -s http://mycluster:8184 testvm
+$ virtctl console --kubeconfig=$KUBECONFIG testvm
```

Comment:
This probably doesn't matter, but in https://github.com/kubevirt/kubevirt/pull/636 virtctl will automatically get the config path from the KUBECONFIG var without needing it passed as an arg to virtctl.

Reply:
Yes we should reflect that, once we have 0.2.1 released. Right now it confused people that kubectl worked but virtctl did not.